crates/ef-testing/src/models/case.rs

```diff
@@ -52,7 +52,7 @@ async fn handle_pre_state(
         eoa_class_hash,
         kakarot.contract_account_class_hash,
     );
-    write_test_state(pre_state, kakarot_address, class_hashes, &mut starknet)?;
+    write_test_state(pre_state, kakarot_address, &class_hashes, &mut starknet)?;
```

Comment:
this is clippy saying we're passing by value but not consuming
only by reference is enough,
less is more